### DISCUSSION TITLE
2022-33: 电脑坏了之后

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#377] [Announcements]
# 2022-33: 电脑坏了之后

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-33/

**Comment by lilydjwg** [comment]
摸摸漩涡的电脑～
这种送修，我会考虑把硬盘拆下来以避免意外和安全隐患。容灾的话，因为我是全系统的备份，给我设备和一个下午，我就可以满血复活了。
我曾经工作的电脑也有故障，每天随机死机一两次，但是不知道向谁反馈才会有人理唉。

**Reply by Xuanwo** [reply]
原来可以自己拆下来再送修啊，我一直以为拆了就不能保修什么😂

**Reply by lilydjwg** [reply]
我也不知道啊，不过我主系统的硬盘是我自己装上去的啊。

**Comment by sunng87** [comment]
去年这款 R7000P 装 Linux 会有什么驱动问题吗？

**Reply by Xuanwo** [reply]
我装的是 Archlinux，最开始的时候因为网卡驱动还没有进内核，相对折腾一些，现在已经 OK 啦

**Comment by lizheming** [comment]
typo Lenovo →_→

**Reply by Xuanwo** [reply]
2333，修这个 typo 的 commit 也打了个 typo

**Comment by ian-qiu** [comment]
自己攒的电脑也遇到开机不久cpu温度过高的问题（啥也没做），怎么也没想到是cpu散热坏了，就差把主板拆了。。。
